lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -715,16 +714,15 @@ class BarTouchTooltipData with EquatableMixin {
         tooltipHorizontalOffset = tooltipHorizontalOffset ?? 0,
         maxContentWidth = maxContentWidth ?? 120,
         getTooltipItem = getTooltipItem ?? defaultBarTooltipItem,
+        getTooltipColor = getTooltipColor ??
+            ((BarChartGroupData? group) => Colors.blueGrey.darken(15)),
```

Comment:
To keep consistency, please use a defined function just like `defaultLineTooltipColor` and `defaultScatterTooltipColor` here

Reply:
Done